### PR TITLE
Sample: captureMessage on a Flutter web plugin

### PR DESCRIPTION
**Pull request by marandaneto**
## :loudspeaker: Type of change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring

## :scroll: Description
Sample: captureMessage on a Flutter web plugin

## :bulb: Motivation and Context
We'd like to have a sample on how to call Sentry from a Flutter web plugin

## :green_heart: How did you test it?
running on Chrome

## :pencil: Checklist

- [X] I reviewed submitted code
- [ ] I added tests to verify changes
- [X] All tests passing
- [X] No breaking changes

### CODE REVIEW

File: flutter/example/lib/example_plugin_web.dart [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+import 'package:flutter_web_plugins/flutter_web_plugins.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+
+/// A web implementation of the SentryFlutter plugin.
```
Comment (marandaneto) [inline flutter/example/lib/example_plugin_web.dart]:
```suggestion
```

File: flutter/example/lib/example_plugin_web.dart [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    final pluginInstance = ExamplePluginWeb();
+    channel.setMethodCallHandler(pluginInstance.handleMethodCall);
+  }
+
+  /// Handles method calls over the MethodChannel of this plugin.
+  /// Note: Check the "federated" architecture for a new way of doing this:
+  /// https://flutter.dev/go/federated-plugins
+  Future<dynamic> handleMethodCall(MethodCall call) async {
+    if (call.method == 'console.log') {
+      Sentry.captureMessage(call.toString());
```
Comment (rxlabz) [inline flutter/example/lib/example_plugin_web.dart:24]:
```suggestion
      Sentry.captureMessage(call.arguments.toString());
```

Comment (marandaneto) [reply flutter/example/lib/example_plugin_web.dart:24]: `call.toString()` does the interpolation of `method` + `arguments` so I'd rather keep as it is

**Review by rxlabz — APPROVED** [review]
LGTM

**Pull request merged**